plugins/generator-1.19.2/forge-1.19.2/tool.definition.yaml

```diff
@@ -1,3 +1,8 @@
+global_templates:
+  - template: elementinits/shield_properties.java.ftl
+    condition: ${w.hasShields()}

```

Comment:
w is the default object for global conditions so you can and should use `hasShields()` directly instead of using FTL expression here, applies to both versions